examples/utils/example-utils/api-report/example-utils.api.md

```diff
@@ -46,6 +45,15 @@ export interface IDetachedModel<ModelType> {
     model: ModelType;
 }
 
+// @internal (undocumented)
```

Comment:
I know these are all internal, but can we document them while we're here?

Reply:
Documented in latest - also removed the const entirely.